frontend/src/libs/components/icon-button/icon-button.tsx

```diff
@@ -0,0 +1,24 @@
+import { Icon } from '#libs/components/icon/icon.js';
+import { type IconName } from '#libs/enums/enums.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  icon: IconName;
+  onClick?: () => void;
+  type?: 'button' | 'submit';
+};
+
+const IconButton: React.FC<Properties> = ({
```

Comment:
Let's get rid of it. So we have a button component. It may contain only icon

Reply:
<img width="480" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/90299049/4cd65c45-e4b4-4fcc-b74a-38617468b406">

@fshabanov suggested the creation of the component.

Reply:
But we can reuse existing one, Button